sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/CdnSampleTests.java

```diff
@@ -6,10 +6,16 @@
 import com.azure.resourcemanager.cdn.samples.ManageCdnWithCustomDomain;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
 
 public class CdnSampleTests extends SamplesTestBase {
 
+    // Test is currently disabled on Linux as a Java API call within this test, source code and not test code,
+    // results in an UnknownHostException:
+    // https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/AppServiceDomainImpl.java#L87
```

Comment:
Should we create GH issue and assign to RM owners?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/30229

Also updated the comment to reference this.